core/src/main/scala/com/databricks/labs/remorph/generators/sql/LogicalPlanGenerator.scala

```diff
@@ -152,10 +152,11 @@ class LogicalPlanGenerator(
   }
 
   // @see https://docs.databricks.com/en/sql/language-manual/sql-ref-syntax-qry-select-lateral-view.html
-  private def lateral(ctx: GeneratorContext, lateral: ir.Lateral): String = lateral match {
-    case ir.Lateral(ir.TableFunction(fn), isOuter) =>
+  private def lateralView(ctx: GeneratorContext, lateral: ir.Lateral): String = lateral match {
```

Comment:
We should rename that back to `lateral` for consistency.

Reply:
done